tcpServ.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+)
+
+func handleConnection(conn net.Conn) {
+	scanner := bufio.NewScanner(conn)
+	for scanner.Scan() {
+		conn.Write([]byte(multipliedInput(scanner.Text())))
+	}
+	if err := scanner.Err(); err != nil {
+		fmt.Errorf("Scanning error", scanner.Err())
+	}
+
+}
+
+func multipliedInput(input string) string {
```

Comment:
Function name does not represent function purpose

Reply:
From my point of view you can just write logic in handleConnection and remove multipliedInput function at all